content/english/covid-distance-checker-wip/_index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "micro:bit — Wireless Social Distancing"
+description: "A guide to keeping at least 2 m apart with Bluetooth"
+date: 2020-07-297T08:45:08-07:00
+prereq: "None"
+difficulty: "Beginner"
+download: ""
+draft: true
```

Comment:
Since this is a draft, let's rename the folder as `covid-distance-checker` (without wip), so that once it's not a draft, the URL will be formatted properly.